src/app/ASKS/menu/index.js

```diff
@@ -22,7 +28,13 @@ const AsksMenu = ({ questionCode }) => {
           <FontAwesomeIcon size="lg" icon={icons[questionCode]} />
           {!isMobile && (
             <HStack spacing={1}>
-              <Text fontSize="xs">{labels[questionCode]}</Text>
+              <Text fontSize="xs">
+                {realm === 'mentormatch' &&
+                userType === 'AGENT' &&
+                questionCode === 'QUE_QUICK_ADD_ITEMS_GRP'
+                  ? labels['QUE_ADD_ITEMS_GRP']
+                  : labels[questionCode]}
```

Comment:
Cannot do this. This is hardcoding and the wrong approach to do it. All of this should be handled from the backend.